llvm/lib/Target/RISCV/MCTargetDesc/RISCVELFStreamer.cpp

```diff
@@ -152,6 +156,16 @@ void RISCVELFStreamer::emitMappingSymbol(StringRef Name) {
   Symbol->setBinding(ELF::STB_LOCAL);
 }
 
+void RISCVELFStreamer::changeISAMappingSymbol(unsigned Attribute,
```

Comment:
This is a bad interface; it says "changeISAMappingSymbol" but takes an ELF attribute and its value that may or may not be for the ISA string.